notifier/chatters.rst

```diff
@@ -218,6 +218,92 @@ to add `message options`_::
 
     $chatter->send($chatMessage);
 
+Adding text to a Microsoft Teams Message
+-----------------------------------------
+
+With a Microsoft Teams, you can use the simple ChatMessage::
+
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\MicrosoftTeamsTransport;
+    use Symfony\Component\Notifier\Message\ChatMessage;
+
+    $chatMessage = (new ChatMessage('Contribute To Symfony'))->transport('microsoftteams');
+    $chatter->send($chatMessage);
+
+The result will be something like:
+
+.. image:: /_images/notifier/microsoft_teams/message.png
+   :align: center
+
+Adding Interactions to a Microsoft Teams Message
+-----------------------------------------
+
+With a Microsoft Teams Message, you can use the
+:class:`Symfony\\Component\\Notifier\\Bridge\\MicrosoftTeams\\MicrosoftTeamsOptions` class
+to add `MessageCard options`_::
+
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action\ActionCard;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action\HttpPostAction;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action\Input\DateInput;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action\Input\TextInput;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\MicrosoftTeamsOptions;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\MicrosoftTeamsTransport;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\Field\Fact;
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\Section;
+    use Symfony\Component\Notifier\Message\ChatMessage;
+
+    $chatMessage = new ChatMessage('');
```

Comment:
Should the first argument really be empty?

Reply:
`new ChatMessage('') ` requires a subject string argument, if this argument is given, then `->text('Text')`  should be skipped while setting `new MicrosoftTeamsOptions()`. In otherw words `new ChatMessage('') ` and `->text('Text')` are equivalents. In my example I used second one to set the text.